tests/conftest.py

```diff
@@ -372,9 +359,10 @@ def upload_callback(request):
             "visibility": "private",
         },
         "POST::api/v1/samples/confirm_upload": "",
-        "GET::api/v1/samples/init_multipart_upload": {
+        "POST::api/v1/samples/init_multipart_upload": {
             "callback_url": "/api/import_file_from_s3",
             "file_id": "abcdef0987654321",
+            "paired_end_file_id": "abcdefg0987654321",
```

Comment:
_May_ be worth mocking `Object` in `boto3` and double checking that _these_ are the keys that get created.